app/controllers/idv/hybrid_mobile/socure/document_capture_controller.rb

```diff
@@ -17,6 +17,14 @@ def show
           Funnel::DocAuth::RegisterStep.new(document_capture_user.id, sp_session[:issuer]).
             call('hybrid_mobile_socure_document_capture', :view, true)
 
+          document_capture_session = DocumentCaptureSession.find_by(
+            uuid: document_capture_session_uuid,
+          )
+          if document_capture_session.socure_docv_capture_app_url.present?
+            @url = document_capture_session.socure_docv_capture_app_url
```

Comment:
since we've added the interstitial page ... it appears we no longer need the other instance variables ... as opposed to cleaning up here. however, i think it'd be easier to rid of them in [LG-14279](https://cm-jira.usa.gov/browse/LG-14279)